Src/Sarif.PatternMatcher.Cli/Program.cs

```diff
@@ -87,5 +88,37 @@ private static bool IsValidVerbName(string verb)
                 verb == "export-search-definitions" ||
                 verb == "import-analyze";
         }
+
+        private static string[] RewriteArgs(string[] args)
+        {
+            bool hasObsoleteArgument = false;
+            bool hasCurrentFileSizeArg = false;
+
+            for (int i = 0; i < args.Length; i++)
+            {
+                if (args[i] == "--file-size-in-kb")
+                {
+                    hasObsoleteArgument = true;
+                    args[i] = "--max-file-size-in-kb";
```

Comment:
>max-file-size-in-kb [](http://example.com/codeflow?start=33&length=19)

we have some md files that shows how to use this. Can you also update to use the new argument, please?

Reply:
I only found one of these and updated accordingly.